### PR TITLE
Add Grcm38/mm10 when looking for mixed ref genomes

### PR DESCRIPTION
**Pull request by oplantalech**
PR #3198 adds a warning when selecting studies from mixed genomes OR if a study with a non-recognized genome is loaded. This PR adds the `Grcm38/mm10` (mouse build) as part of the "recognized" genomes so that the warning is removed from these studies.

### CODE REVIEW

File: src/shared/lib/referenceGenomeUtils.tsx [hunk 1]
```diff
@@ -21,8 +25,11 @@ export function isMixedReferenceGenome(studies: CancerStudy[]): boolean {
     const isAllStudiesGRCh38 = _.every(studies, study => {
         return isGrch38(study.referenceGenome);
     });
+    const isAllStudiesGRCm38 = _.every(studies, study => {
+        return isGrcm38(study.referenceGenome);
+    });
     // return true if there are mixed studies
-    return !(isAllStudiesGRCh37 || isAllStudiesGRCh38);
+    return !(isAllStudiesGRCh37 || isAllStudiesGRCh38 || isAllStudiesGRCm38);
```
Comment (SRodenburg) [inline src/shared/lib/referenceGenomeUtils.tsx:32]: ~~These condition looks strange to me. I didn't test this, but shouldn't it be:
```(!isAllStudiesGRCh37 || !isAllStudiesGRCh38 || !isAllStudiesGRCm38)```~~

I think its fine

**Review by SRodenburg — APPROVED** [review]
Since you are merely copying the functions of existing code, it should be OK.

**Review by leexgh — APPROVED** [review]
Thanks! Probably good to also update the warning message for mixed reference genome at some point.
https://github.com/cBioPortal/cbioportal-frontend/pull/3467/files#diff-a136b7727e9272e176041370c1eee92fa1b806b7cba6ebfa6eda64a5a718a4baR40-R44

**Comment by Sjoerd-van-Hagen** [comment]
@inodb can we get this merged (after rebase?)

**Comment by inodb** [comment]
@oplantalech not sure what the issue is, but there seems to be some problem for IE11. That one doesn't fail on master branch. The changes here look pretty innocuous so not sure what the issue is

**Comment by Sjoerd-van-Hagen** [comment]
@inodb do you have an error message? We also saw some weird error on IE with one of our clients. We were not using this branch though.

**Comment by inodb** [comment]
@Sjoerd-van-Hagen Just circleci error log: https://app.circleci.com/pipelines/github/cBioPortal/cbioportal-frontend/2376/workflows/c6bfdbea-ef2c-4f6c-ae1e-7cf041c54c42/jobs/85267. Might have to reproduce in e.g. browserstack to see exactly what's going on. Could be unrelated to the PR, but if u check master branch the ie11 test doesn't seem to fail much: https://app.circleci.com/pipelines/github/cBioPortal/cbioportal-frontend?branch=master

**Comment by inodb** [comment]
@Sjoerd-van-Hagen hmm actually I see it fail on this PR now as well: https://github.com/cBioPortal/cbioportal-frontend/pull/3476. So guess it is unrelated

**Comment by inodb** [comment]
@Sjoerd-van-Hagen The master branch is a little unstable atm cause we're in the midst of v3.4.18 release and ran into some issues. Will try and stabilize and merge this

**Pull request merged**